tavern/schemas/tests.schema.yaml

```diff
@@ -156,8 +156,18 @@ schema;stage:
           required: false
           func: validate_timeout_tuple_or_float
 
+        cert:
+          func: validate_extensions
+          type: seq
+          required: false
+          sequence:
+          - type: str
+            required: true
+          - type: str
+            required: true
+
         verify:
-          type: bool
+          type: any
```

Comment:
If you change this to 'any' then you should put some validation in `extensions.py` to make sure that it is a boolean or a string, which are the only types accepted by requests for this keyword